Hangfire.Extensions.ApplicationInsights/ApplicationInsightsBackgroundJobPerformer.cs

```diff
@@ -20,29 +20,31 @@ public class ApplicationInsightsBackgroundJobPerformer : IBackgroundJobPerformer
 
         public object Perform(PerformContext context)
         {
-            var requestTelemetry = new RequestTelemetry()
+            var dependencyTelemetry = new DependencyTelemetry()
             {
                 Name = $"JOB {context.BackgroundJob.Job.Type.Name}.{context.BackgroundJob.Job.Method.Name}",
             };
 
+            dependencyTelemetry.Context.Operation.Id = context.GetJobParameter<string>("operationId");
+            dependencyTelemetry.Context.Operation.ParentId = context.GetJobParameter<string>("operationParentId");
 
             // Track Hangfire Job as a Request (operation) in AI
```

Comment:
Should be
// Track Hangfire Job as a Dependency (operation) in AI

Reply:
Thanks